docs/api/assertions.md

```diff
@@ -101,12 +101,14 @@ We can declare these validation checks through `dspy.Suggest` statements (as we
 dspy.Suggest(
     len(query) <= 100,
     "Query should be short and less than 100 characters",
+    target_module=self.generate_query
```

Comment:
Wait I don't think we want the examples to be verbose like this. It's nice that they were implicit some of the time.